numpy/lib/polynomial.py

```diff
@@ -1017,7 +1017,7 @@ def polydiv(u, v):
     (array([1.5 , 1.75]), array([0.25]))
 
     """
-    truepoly = (isinstance(u, poly1d) or isinstance(u, poly1d))
```

Comment:
I think this was a typo and the second `u` should be `v`

Reply:
This would indeed change the comparison. Shall it change it to `(isinstance(u, poly1d) or isinstance(v, poly1d))` ?

Reply:
Yes please.